File: src/app/lead-dto.ts
export class LeadDTO {

    firstname:string;
	lastname:string;
    lastcontacted:string;
	product:string;
	email:string;
	contact:string;
	leadStage:string;


	constructor(
	firstname:string, 
	lastname:string,
	lastcontacted:string,
	product:string,
	email:string,
	contact:string,
	leadStage:string
	){

		this.firstname=firstname;
		this.lastname=lastname;
		this.lastcontacted=lastcontacted;
		this.product=product;
		this.email=email;
		this.contact=contact;
		this.leadStage=leadStage;

		}

}

File: java/com/cg/hbm/dto/LeadDTO.java
package com.cg.hbm.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor

public class LeadDTO {
	
	private int id;
	
	private String firstname;
	private String lastname;
	private String lastcontacted;
	private String product;
	private String email;
	private String contact;
	private String leadStage;
	

}
